content/using/running/minio.md

```diff
@@ -0,0 +1,176 @@
++++
+title = "Self-hosting S3 Storage"
+description = "How to set up and self-host MinIO S3 storage"
+template = "doc.html"
+weight = 2
+[extra]
+hidetitle = "true"
++++
+
+Adding [S3](https://aws.amazon.com/s3/) storage to Urbit unlocks some great new features, such as the ability to post media to chats and upload custom avatars. This is a guide to self-hosting [MinIO](https://min.io), an S3 compatible block storage solution.
```

Comment:
The way this is written, someone might get the impression that you need S3 to be able to post media etc. This is not the case - you can post media located anywhere. Instead you should say something about getting the ability to automatically upload media by e.g. dragging and dropping.